test/tokens.test.js

```diff
@@ -45,7 +45,7 @@ function assertObject(b, t, obj, k, v) {
 ///--- Tests
 
 before(function (cb) {
-    //this.bucket = 'moray_unit_test_' + uuid.v4().substr(0, 7);
+    //this.bucket = 'moray_unit_test_' + uuidv4().substr(0, 7);
```

Comment:
Can we just drop this line since it's commented out and then drop the `require('uuid/v4')` from this module?

Reply:
Sure, done in latest commit.